src/serlio/modifiers/PRTModifierAction.cpp

```diff
@@ -866,7 +917,8 @@ MStatus PRTModifierAction::createNodeAttributes(const RuleAttributeSet& ruleAttr
 						auto tryParseRangeAnnotation = [](const prt::Annotation* an) -> std::pair<double, double> {
 							auto minMax = std::make_pair(std::numeric_limits<double>::quiet_NaN(),
 							                             std::numeric_limits<double>::quiet_NaN());
-							for (int argIdx = 0; argIdx < an->getNumArguments(); argIdx++) {
+							const size_t numArgs = an->getNumArguments();
+							for (int argIdx = 0; argIdx < numArgs; argIdx++) {

```

Comment:
minimize scope of `numArgs`:

```suggestion
							for (int argIdx = 0, const size_t numArgs = an->getNumArguments(); argIdx < numArgs; argIdx++) {

```

Reply:
numArgs is used a few lines under the for loop.. 
Technically the if numArgs == 2 could be removed since it's checked in getRangeType, but it seemed logical to sanity check it if the variable is computed anyways.. 
How would you like to proceed?